src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -317,11 +317,9 @@ private void handleError(@Nonnull final String errorMessage, @Nullable final Thr
      * @param draft draft containing data that could differ from data in {@code entry}.
      * @param updateActions the update actions to update the {@link InventoryEntry} with.
      */
-    private void handleError(@Nonnull final String errorMessage, @Nullable final Throwable exception,
+    private void handleError(@Nonnull final SyncException syncException,
         final int failedTimes, @Nullable final InventoryEntry entry, @Nullable final InventoryEntryDraft draft,
```

Comment:
failedTimes is always 1, you can remove the param and just call it with 1 value.

Reply:
4463c8e6c39d567d201182557947d6c283cc691f